pom.xml

```diff
@@ -48,35 +47,53 @@
         </license>
     </licenses>
 
-    <dependencyManagement>
-        <dependencies>
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-all</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-core</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-api</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-            <dependency>
-                <!-- this gives us flexible and easy-to-use logging; just edit logback-custom.xml! -->
-                <groupId>org.apache.brooklyn</groupId>
-                <artifactId>brooklyn-logback-xml</artifactId>
-                <version>${brooklyn.version}</version>
-            </dependency>
-
-        </dependencies>
-    </dependencyManagement>
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-all</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-core</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-api</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.cloudfoundry</groupId>
+            <artifactId>cloudfoundry-client-lib</artifactId>
+            <version>${cloudfoundry.client.version}</version>
+        </dependency>
+
+        <!-- http://mvnrepository.com/artifact/org.mockito/mockito-all -->
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-all</artifactId>
+            <version>${mockito.version}</version>
+        </dependency>
+
+        <dependency>
+            <!-- this gives us flexible and easy-to-use logging; just edit logback-custom.xml! -->
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-logback-xml</artifactId>
+            <version>${brooklyn.version}</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.apache.brooklyn</groupId>
+            <artifactId>brooklyn-core</artifactId>
+            <version>${brooklyn.version}</version>
+            <classifier>tests</classifier>
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
 
     <build>
```

Comment:
I have remove the version of the plugin because, as you have mentioned, it is not necessay